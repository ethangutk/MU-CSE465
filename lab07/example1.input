z = 3 + (4.3*7**2 - 2)
z
y = -3.141*z^
y