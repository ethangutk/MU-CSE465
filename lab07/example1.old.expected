ID             	z
EQUALS         	=
REAL           	3
OP_PLUS        	+
LPAREN         	(
REAL           	4.3
OP_MULTIPLY    	*
REAL           	7
OP_EXPONENT    	**
REAL           	2
OP_MINUS       	-
REAL           	2
RPAREN         	)
ID             	z
ID             	y
EQUALS         	=
REAL           	-3.141
OP_MULTIPLY    	*
ID             	z
Invalid character '^' at line 3 column 13
